src/event-handler/index.ts

```diff
@@ -45,3 +47,13 @@ export function createEventHandler(options: Options<any>): EventHandler {
     receive: receive.bind(null, state),
   };
 }
+
+export function createLogger(logger?: Partial<Logger>) {
```

Comment:
I think it would be better if this lived in its own file somewhere, as the some of the functions are marked as being usable standalone and this is creating an implicit dependency on `event-handler`.

Having it in its own file means it won't trigger any import side-effects and make it easier to pull the standalones into their own packages if we decide to go that way in the future.

Reply:
Currently there isn't a problem pulling the standalones into their own packages or triggering import side effects because everything that imports `createLogger()` already imports `createEventHandler()`, which is why I thought it best to put it here. I can imagine that changing in the future, but I think my choice would be to cross that bridge when we come to it?

Reply:
I still would have preferred to have had this somewhere else, as it would make it easier to test in isolation and be more inline with the single responsibility principle.